Employee.java

```diff
@@ -0,0 +1,65 @@
+public class Employee implements Comparable<Employee> {
+    private int ID;
+    private String NAME;
+    private String Position;
+    private int Salary;

```

Comment:
should start with lowercase, e.g.: `position`, `salary`

Reply:
fixed on cac633a